manifests/vrrp/unicast_peer.pp

```diff
@@ -0,0 +1,23 @@
+# == Define: keepalived::vrrp::unicast_peer
+#
+# === Parameters:
+#
+# $instance::   Name of vrrp instance this peer belongs to
+#
+# $ip_address:: IP address of unicast peer
+#               Default: $name
+#
```

Comment:
instead of this old javadoc? syntax, can you please use puppet-strings style to document the defined resource?

Reply:
Ah, you'd asked for this on the other PR, and I didn't understand for some reason, so I just tried to make the documentation look as much as possible like other classes and defined types in this module. It's now done in puppet-strings format, but without other examples in this module, I don't have a sense of whether it meets our standards. I saw that https://voxpupuli.org/docs/reviewing_pr/ mentions examples and params, so there are those.